gennet-module/gennet.py

```diff
@@ -280,12 +280,12 @@ def generate_toml(traits_dir, topics, traits_list):
         topic_str = f"\"{topic_str}\""
 
     for trait in traits_list[1:]:       # skip the first trait as it is docker/node selector.
+        toml = f'#{trait}\n'
         with open(f"{traits_dir}/{trait}.toml", 'rb') as f:
-            toml = ""
-            for key, value in tomli.load(f).items():
-                toml += f"{key} = {str(value)}\n"
-        tomls += toml
-    return f"{tomls}topics = {topic_str}\n"
+            strlines = [l.decode("utf-8").strip() for l in f if not len(l.split()) == 0]
+            toml += ''.join([f'{l}\n' for l in strlines if not l.startswith('#')])
```

Comment:
And this is another example of why we need tests. A simple test with comment lines in the toml would make us detect this.